kibana/tests/kibana_test.py

```diff
@@ -333,3 +333,11 @@ def test_override_the_serverHost():
     c = r['deployment'][name]['spec']['template']['spec']['containers'][0]
     assert c['env'][1]['name'] == 'SERVER_HOST'
     assert c['env'][1]['value'] == 'localhost'
+
+  def test_adding_pod_annotations():
```

Comment:
There is some extra whitespace here which causes python to not run.

Reply:
Fixed